sandbox.sh

```diff
@@ -130,7 +147,7 @@ EOF
     --burn-cap 2 \
     --force
 
-  for VALIDATOR in ${VALIDATORS[@]}; do
+  for VALIDATOR in "${VALIDATORS[@]}"; do
```

Comment:
What's the reason of adding these?

Reply:
While I was in there I fixed some shellcheck stuff. We should probably add it to CI